plugins/system/shortcut/shortcut.php

```diff
@@ -0,0 +1,34 @@
+<?php
+// no direct access
+defined( '_JEXEC' ) or die;
+
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\CMS\Language\Text;
+
+/**
+ * Skipto plugin to add accessible keyboard navigation to the site and administrator templates.
+ *
+ * @since  4.1
```

Comment:
```suggestion
 * @since  __DEPLOY_VERSION__
```
always use __DEPLOY_VERSION__ instead of the version because this will automatically changed on release